static/src/javascripts/projects/common/modules/commercial/creatives/hosted-thrasher-multi.js

```diff
@@ -0,0 +1,53 @@
+define([
+    'common/utils/fastdom-promise',
+    'common/utils/config',
+    'common/utils/template',
+    'common/modules/commercial/creatives/add-tracking-pixel',
+    'text!common/views/commercial/creatives/hosted-thrasher-multi.html'
+], function (
+    fastdom,
+    config,
+    template,
+    addTrackingPixel,
+    hostedThrasherStr
+) {
+    var hostedThrasherTemplate;
+
+    var HostedThrasherMulti = function ($adSlot, params) {
+        this.$adSlot = $adSlot;
+        this.params = params;
+    };
+
+    HostedThrasherMulti.prototype.create = function () {
+        hostedThrasherTemplate = template(hostedThrasherStr);
+
+        fastdom.write(function () {
```

Comment:
can you please return the promise and resolve it to `true`? this is so that we can track the rendering (i'll be raising a pr about that soon)